app/routes.js

```diff
@@ -327,7 +333,6 @@ export default (config) => {
                   content: TripQueries,
                   meta: RouteQueries,
                 }}
-                render={{ title: ({ props, element }) => React.cloneElement(element, props) }}
```

Comment:
Removing these affect what is shown when moving between route and stop pages. We probably should design what/how to show during the loading of data.